### DISCUSSION TITLE
Receiving and playing?

### DISCUSSION THREAD

**Discussion by wusspuss** [aiortc/aiortc#915] [Q&A]
The examples seem to only demonstrate how to send video from Python and play it in the browser, or receive it in Python and save to file. Can you please recommend how best to actually play it from Python as it comes? I figure I can get pyav's `AudioFrame` and `VideoFrame`, but I don't see a way to actually play them, neither with in this library's documentation nor pyav's. I'm asking here just in case this library might have a higher level interface for playing than "manually" working  with `AudioFrame`s?

**Comment by MrPh0enix** [comment]
I was thinking of a solution for the same. I tried displaying frame by frame using cv2.imshow but it didn't work (the cv2 window just stopped responding). Then I tried inducing a 1 second delay between each imshow call, and it still crashed. Finally I used matplotlib combined with the 1 sec delay and it displayed the frames. But this method isnt very practical.
